shell/platform/windows/win32_window.cc

```diff
@@ -153,13 +153,34 @@ Win32Window::MessageHandler(HWND hwnd,
         window->OnScroll(
             0.0, -(static_cast<short>(HIWORD(wparam)) / (double)WHEEL_DELTA));
         break;
+      case WM_UNICHAR: {
+        // Tell third-pary app, we can support Unicode.
+        if (wparam == UNICODE_NOCHAR)
+          return TRUE;
```

Comment:
If we return true, doesn't that mean we need to actually handle all the non-UNICODE_NOCHAR cases here by sending the character through?

Reply:
Processing flow:
1. The third-party application wants to send UNICODE to us, send this parameter(UNICODE_NOCHAR) to ask us, and we return TRUE to indicate that we can handle it. If we give it to DefWindowProc for processing, the default is to return FALSE.
2. When the third-party application gets the TRUE, it sends the UNICODE, and we give it to DefWindowProc, which converts it to one or two WM_CHAR and sends it to us.

In fact, few third-party applications use this message anymore. Most of them send WM_CHAR directly.

Reply:
Hm, that's inconsistent with what's described [here](https://blog.keyman.com/2011/06/accepting-unicode-input-in-your-windows-application/), which is one of the things that turned up when I was trying to find a usable description of how this works (since the Win32 description doesn't seem to make sense at all). Since it's third-party only, maybe the behaviors of the applications themselves are inconsistent?

We can leave this as-is for now and revisit if there are problems (especially since it's not actually clear to me that this will *ever* be used if the Runner's window is created in Unicode mode).